### DISCUSSION TITLE
SpOC 3 has now been concluded.

### DISCUSSION THREAD

**Discussion by maxbannach** [esa/SpOC3#14] [Announcements]
We have extracted the results for 30 June, end of date, AoE, and are currently generating the final ranking. The submission system on Optimize remains open (and the scoreboard on Optimize will continue to change, but with no effect on the SpOC results).

The results and final ranking will be published soon. There will also be a presentation at GECCO in two weeks, which will present the competition and the final leaderboard.

Thank you all for your participation in this year's competition. Without your involvement, it would not be possible. 🚀

**Comment by ortegga** [comment]
Thank you for the organization and the fun ! Cheers to all fellow competitors too :)

**Comment by neinecke** [comment]
Thank you for this great competition. It sure was a lot of work to prepare and it was again a lot of fun contributing :)

**Comment by dietmarwo** [comment]
Thanks a lot, this time the tasks were a lot of fun to solve.
